.github/workflows/test.yml

```diff
@@ -9,7 +9,42 @@ on:
     types:
```

Comment:
Do we want to still call this test.yml or would another name like workflows.yml work?

Reply:
Maaaayyybbeee.
I'm not sure what usefully encapsulates the purpose of the file.

`review`, with `Automated Review Pipeline` as the title?

Reply:
Done